server/models/user.py

```diff
@@ -1,21 +1,27 @@
 import logging
 
-from sqlalchemy import Column, Integer, String, ForeignKey, Table, UniqueConstraint, CheckConstraint, select
+from sqlalchemy import Column, Integer, String, ForeignKey, UniqueConstraint, CheckConstraint, select
 from sqlalchemy.orm import relationship
 
 from server.models import Base
+from server.models.chat import chat_to_user
 
 
 log = logging.getLogger(__name__)
 
 
-friendship = Table(
```

Comment:
Why do you decide to create class FriendSip? Original table solves the same problem and work by the same way.
At the same time for ChatToUser table you just create a table without explicit class.

Reply:
We need id of a friendship to use it in Message model. Association table that we use before had no id field.

Reply:
Just have removed  friendship from Message, but prefer to leave it as class for the current moment.
May be we will return to this question later.